integration/droplet_backups_test.go

```diff
@@ -79,7 +79,8 @@ ID      Name     Type        Distribution    Slug      Public    Min Disk
      "distribution": "Fedora",
      "type": "snapshot",
      "slug": "slimey",
-     "min_disk_size": 25
+     "min_disk_size": 25,
+	 "created_at": "2018-09-20T19:28:00Z"
```

Comment:
A small nitpick change we could do. Let's change the tabs with spaces? :pray: 
```suggestion
     "created_at": "2018-09-20T19:28:00Z"
```